2022-10-11 04:24:22,103 INFO    Thread-2  :28092 [wandb_setup.py:_flush():75] Loading settings from /root/.config/wandb/settings
2022-10-11 04:24:22,103 INFO    Thread-2  :28092 [wandb_setup.py:_flush():75] Loading settings from wandb/settings
2022-10-11 04:24:22,103 INFO    Thread-2  :28092 [wandb_setup.py:_flush():75] Loading settings from environment variables: {'project': 'meme_experiments', 'entity': 'vietnguyen', 'root_dir': '/content/SSLMemes', 'sweep_id': 'bp7l5vlp', 'run_id': '7341ilm5', 'sweep_param_path': '/content/SSLMemes/wandb/sweep-bp7l5vlp/config-7341ilm5.yaml'}
2022-10-11 04:24:22,103 INFO    Thread-2  :28092 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'src/train.py', 'program': '/content/SSLMemes/src/train.py'}
2022-10-11 04:24:22,103 INFO    Thread-2  :28092 [wandb_init.py:_log_setup():405] Logging user logs to /content/SSLMemes/wandb/run-20221011_042422-7341ilm5/logs/debug.log
2022-10-11 04:24:22,103 INFO    Thread-2  :28092 [wandb_init.py:_log_setup():406] Logging internal logs to /content/SSLMemes/wandb/run-20221011_042422-7341ilm5/logs/debug-internal.log
2022-10-11 04:24:22,103 INFO    Thread-2  :28092 [wandb_init.py:init():439] calling init triggers
2022-10-11 04:24:22,103 INFO    Thread-2  :28092 [wandb_init.py:init():443] wandb.init called with sweep_config: {'batch_size': 48, 'dropout': 0.5, 'epochs': 1, 'hidden_size': 256, 'learning_rate': 0.03686021362339303, 'optimizer': 'sgd'}
config: {}
2022-10-11 04:24:22,103 INFO    Thread-2  :28092 [wandb_init.py:init():492] starting backend
2022-10-11 04:24:22,103 INFO    Thread-2  :28092 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-10-11 04:24:22,120 INFO    Thread-2  :28092 [backend.py:ensure_launched():219] starting backend process...
2022-10-11 04:24:22,130 INFO    Thread-2  :28092 [backend.py:ensure_launched():225] started backend process with pid: 28137
2022-10-11 04:24:22,132 INFO    Thread-2  :28092 [wandb_init.py:init():501] backend started and connected
2022-10-11 04:24:22,141 INFO    Thread-2  :28092 [wandb_run.py:_config_callback():992] config_cb None None {'batch_size': 48, 'dropout': 0.5, 'epochs': 1, 'hidden_size': 256, 'learning_rate': 0.03686021362339303, 'optimizer': 'sgd'}
2022-10-11 04:24:22,144 INFO    Thread-2  :28092 [wandb_init.py:init():565] updated telemetry
2022-10-11 04:24:22,171 INFO    Thread-2  :28092 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-10-11 04:24:23,123 INFO    Thread-2  :28092 [wandb_run.py:_on_init():1759] communicating current version
2022-10-11 04:24:23,165 INFO    Thread-2  :28092 [wandb_run.py:_on_init():1763] got version response upgrade_message: "wandb version 0.13.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-10-11 04:24:23,166 INFO    Thread-2  :28092 [wandb_init.py:init():625] starting run threads in backend
2022-10-11 04:24:25,820 INFO    Thread-2  :28092 [wandb_run.py:_console_start():1733] atexit reg
2022-10-11 04:24:25,821 INFO    Thread-2  :28092 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.REDIRECT
2022-10-11 04:24:25,821 INFO    Thread-2  :28092 [wandb_run.py:_redirect():1611] Redirecting console.
2022-10-11 04:24:25,822 ERROR   Thread-2  :28092 [wandb_run.py:_redirect():1670] Failed to redirect.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/wandb_run.py", line 1663, in _redirect
    out_redir.install()
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/usr/lib/python3.7/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-10-11 04:24:25,824 INFO    Thread-2  :28092 [wandb_init.py:init():664] run started, returning control to user process
2022-10-11 04:24:25,825 INFO    Thread-2  :28092 [wandb_run.py:_config_callback():992] config_cb None None {'img_feature_path': 'data/features/lxmert/', 'train_csv_path': 'data/splits/random/memotion_train.csv', 'val_csv_path': 'data/splits/random/memotion_val.csv', 'model_type': 'lxmert', 'model_path': 'unc-nlp/lxmert-base-uncased', 'epoch': 100, 'eval_step': 100, 'amp': True, 'output_dir': './tmp', 'checkpoint_step': 100, 'random_seed': 42, 'resume_training': False, 'semi_supervised': True}
2022-10-11 04:24:26,197 INFO    Thread-2  :28092 [wandb_run.py:_config_callback():992] config_cb None None {'output_hidden': 768}
2022-10-11 04:24:32,525 INFO    Thread-2  :28092 [wandb_run.py:finish():1539] finishing run vietnguyen/meme_experiments/7341ilm5
2022-10-11 04:24:32,525 INFO    Thread-2  :28092 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 1
2022-10-11 04:24:32,527 INFO    Thread-2  :28092 [wandb_run.py:_restore():1674] restore
2022-10-11 04:24:32,533 INFO    Thread-2  :28092 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 699
}

2022-10-11 04:24:32,782 INFO    Thread-2  :28092 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 699
}

2022-10-11 04:24:33,139 INFO    Thread-2  :28092 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 10119
}

2022-10-11 04:24:33,241 INFO    Thread-2  :28092 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 10119
}

2022-10-11 04:24:33,343 INFO    Thread-2  :28092 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10119
  total_bytes: 10119
}

2022-10-11 04:24:33,444 INFO    Thread-2  :28092 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10119
  total_bytes: 10119
}

2022-10-11 04:24:33,546 INFO    Thread-2  :28092 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10119
  total_bytes: 10119
}

2022-10-11 04:24:33,648 INFO    Thread-2  :28092 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10119
  total_bytes: 10119
}

2022-10-11 04:24:33,750 INFO    Thread-2  :28092 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10119
  total_bytes: 10119
}

2022-10-11 04:24:34,239 INFO    Thread-2  :28092 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10119
  total_bytes: 10119
}

2022-10-11 04:24:34,597 INFO    Thread-2  :28092 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10119
  total_bytes: 10119
}
local_info {
}

2022-10-11 04:24:36,156 INFO    Thread-2  :28092 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-10-11 04:24:36,156 INFO    Thread-2  :28092 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-10-11 04:24:36,158 INFO    Thread-2  :28092 [wandb_run.py:_footer_sync_info():2822] logging synced files
